app/models/organization.rb

```diff
@@ -20,14 +20,22 @@ def reportable_orders_item_count
     reportable_orders.map(&:item_count).inject(0) { |a, e| a + e }
   end
 
-  def self.counties
-    Organization.select(:county).map(&:county).uniq
-  end
-
   def primary_address
     addresses.first
   end
 
+  def self.by_county_report
+    results = {}
+    counties.sort_by { |county| (county.presence || "no county").downcase }.each do |county_name|
+      results[county_name.presence || "No County"] = Organization.where(county: county_name)
+    end
+    results
+  end
+
+  def self.counties
+    Organization.select(:county).map(&:county).uniq
```

Comment:
This will produce the same results: `Organization.pluck(:county).uniq` but won't load all the organization objects unnecessarily.